.github/workflows/build.yml

```diff
@@ -0,0 +1,19 @@
+name: Build
+
+on:
+  push:
+    branches: [ main ]
```

Comment:
I'm not sure if we want to be doing this on push, or need to be doing this only on release. If we do it on push, I think we need to be managing our tag more closely.

Reply:
I'm planning to use
`tags: ${{ github.sha }}` at first, to mimic the  `VERSION=$(shell git rev-parse HEAD)` logic currently present in the Makefile.

Long term what I want to be doing is, in case this is a release build take the semver of the release as the image tag.
pending a bit more discussion with @jomkz about how a release looks like (I'm for git tag).

Reply:
I am for git tag as well. However, I'm unsure if there is a reason for us to assign a semver for official relaseas?